### DISCUSSION TITLE
Parameter "externe Aktivierung" im Zeitplan bei Schaltzeiten

### DISCUSSION THREAD

**Discussion by Lausbua** [camueller/SmartApplianceEnabler#469] [Q&A] [answered]
Hallo,

habe vor 2 Tagen meine Installation von 2.1.0 auf 2.2.1 geupdatet. Dabei wurde im Zeitplan der Parameter "externe Aktivierung" gesetzt. Vermutlich hat dadurch mein Zeitplan nicht mehr funktioniert und das Gerät blieb trotz Überschuss inaktiv.
Die Ampel im Status war deaktiviert, keinerlei rot/gelb/grün.

Habe den Parameter deaktiviert. Die Ampel zeigt jetzt soweit eine rote Ampel. Das passt soweit da aktuell der frühestmögliche Einschaltzeitpunkt noch nicht erreicht wurde.

Was macht dieser Parameter? Ich finde dazu nichts oder sehe den Wald vor lauter Bäumen nicht.

![image](https://user-images.githubusercontent.com/3309871/218344135-afa8495f-9174-489b-bdf7-16afe7957c87.png)

**Comment by Lausbua** [comment, marked as answer]
Ohje, habs gefunden.

https://github.com/camueller/SmartApplianceEnabler/issues/384

Ermöglicht den Zeitplan über MQTT-Nachricht enabled true/false zu aktivieren oder deaktivieren.

Eventuell könnte man den Parameter bei Update dann im Zeitplan standardmäßig auf deaktiviert setzen?

**Reply by camueller** [reply]
Das ist standardmaessig dekativiert.

**Reply by camueller** [reply]
Ich habe es auch nochmal getestet: Beim Laden der Konfiguration durch 2.2.1 wird "externe Aktivierung" nicht selbständig aktiviert.
Vermutlich hätte ich in diesem Fall auch Fehlermeldungen von anderen Usern bekommen, da inzwischen immerhin schon > 80 auf die neue Version gewechselt haben.

**Reply by camueller** [reply]
Ich habe die Dokumentation erweitert: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Schedules_DE.md
